GliaWidgets.xcodeproj/project.pbxproj

```diff
@@ -180,7 +180,7 @@
 		1B0E1F137174A56E01875C95 /* Pods_GliaWidgetsTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3280C82C7791B7FAB49280F2 /* Pods_GliaWidgetsTests.framework */; };
 		24A54E58DEFB804A3A90ABEC /* Pods_GliaWidgets.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 43CF5FD712275E8B9AA7311D /* Pods_GliaWidgets.framework */; };
 		39B13FD6530AD29917C878A0 /* Pods_GliaWidgets.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 65D212CF1DC701685125C7B7 /* Pods_GliaWidgets.framework */; };
-		4B1AF277CB06801731BEC963 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		4B1AF277CB06801731BEC963 /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
is this correct?

Reply:
yep, this does `pod install` by some reason.